Makefile

```diff
@@ -32,7 +32,8 @@ help:
 
 check-gopath:
 ifndef GOPATH
-	$(error GOPATH is not set)
+GOPATH := $(shell hack/setup-go.sh .)
+export GOPATH
```

Comment:
Also consider overriding `GOBIN` in this case (or unexporting it) because otherwise binaries may not be installed to `$(GOPATH)/bin` as we expect on line 106